### PR TITLE
Tiff/form styling refactor

### PR DESCRIPTION
**Pull request by cnnmon**
# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
refactoring styling: removed redundant code, redid code hierarchy for components, converted to styled components (as much as possible)

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] above 

## How can the reviewer test your code? 👩‍💻
_example: run `npm start` and click through Navigation bar_ 
run through form flow

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 
n/a besides existing form errors:
1. go back in especially AdditionalQuestionManager is sketch
2. can we remove \n
image

## Resources 📔
_link online resources and related PRs_ 

🧜‍♀️ cc: @shannonbonet @phoebeli23

### CODE REVIEW

**Comment by shannonbonet** [comment]
Note on bugs: 
- could you explain what sketch behavior happens with AdditonalQuestionManager? 
- we can change description texts in Firebase console under the questions collection!

File: src/components/Inputs/Radio/Radio.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  const [value, setValue] = useState(existingAnswer);
+
+  const onChange = (val: any): void => {
+    setValue(val);
+    setAnswer(question, val);
+  };
+
+  const getRadioComponent = (option: any): any => {
+    return (
+      <ButtonContainer>
```
Comment (shannonbonet) [inline src/components/Inputs/Radio/Radio.tsx:22]: getting a React unique key error here - this was fixed in earlier PRs, so make sure you're not overriding the change here
Screen Shot 2021-12-05 at 12 23 06 AM

File: src/components/Inputs/Dropdown/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components/native';
+import { Colors } from 'assets/Colors';
+
+export const ButtonContainer = styled.View`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const PlatformContainer = styled.View`
```
Comment (shannonbonet) [inline src/components/Inputs/Dropdown/styles.tsx:10]: add margin here to match/standardize spacing between description + box of other questions 
Screen Shot 2021-12-05 at 12 24 16 AM

File: src/components/Questions/styles.tsx [hunk 3]
```diff
@@ -18,3 +16,7 @@ export const ButtonView = styled.View`
   flex-direction: row;
   justify-content: center;
 `;
+
+export const Container = styled.ScrollView`
```
Comment (shannonbonet) [inline src/components/Questions/styles.tsx:20]: [PROBABLY WRONG, MIGHT NOT REQUIRE ACTION] This padding might be acting on the scroll bar, not necessarily the page view itself? Might have to do something similar with PageContainer and InnerScroll 
Screen Shot 2021-12-05 at 12 27 13 AM

Comment (shannonbonet) [reply src/components/Questions/styles.tsx:20]: This error doesn't show up for GeneralQuestions though, so I'm a bit confused why this is happening. If I keep submitting the form (below is the 3rd-4th try) the scroll bar keeps moving further center. WHAT IS THIS BUG. Probably not your doing, but documenting the problem here: 
Screen Shot 2021-12-05 at 12 31 13 AM

**Review by shannonbonet — COMMENTED** [review]
Thank you for refactoring everything. You're a god send. Left reviews, mostly make sure you're not overriding fixes made in previous PRs

File: src/components/Inputs/Radio/Radio.tsx (hunk 1 above)
Comment (cnnmon) [reply src/components/Inputs/Radio/Radio.tsx:22]: how was it fixed in earlier PRs?

File: src/components/Questions/styles.tsx (hunk 3 above)
Comment (cnnmon) [reply src/components/Questions/styles.tsx:20]: can you inspect element at all? this doesn't show up on web

File: src/components/Inputs/Radio/Radio.tsx (hunk 1 above)
Comment (shannonbonet) [reply src/components/Inputs/Radio/Radio.tsx:22]: https://github.com/calblueprint/siren/blob/125df27f67f0e371a9c183f397ee594e37d0587a/src/components/Radio/radio.tsx#L51

**Pull request merged**